build_tools/circle/build_doc.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+q#!/usr/bin/env bash
```

Comment:
A small typo was introduced here @NicolasHug, probably exiting vim.

Reply:
Oops! Thanks for noting, I pushed a fix